db/models/AllPlayers.js

```diff
@@ -0,0 +1,16 @@
+import mongoose from "mongoose";
+
+const { Schema } = mongoose;
+
+const allPlayersSchema = new Schema({
+  name: { type: String, required: true },
+  skill: { type: String, required: true },
+  language: [String],
+  gender: { type: String, required: true },
+});
+
+const AllPlayers =
+  mongoose.models.AllPlayers2 ||
+  mongoose.model("AllPlayers2", allPlayersSchema);
+
+export default AllPlayers;
```

Comment:
Why do you call the model `AllPlayers2`? And usually there would be no need to emphasize that this collection is for all players. Without knowing your overall data model I would propose to call it simply `Player` (remember: model names should always be singular, the corresponding collection will then be called `players` if you don't specify differently)

Reply:
I had an AllPlayers schema and collection, with language as string (and not as an array of strings like now).
I didn´t truly understand how the code works, that is why I changed a few things and just observed what happens. I understand now, that the schema is how a single data in the collection should look like, that is why you are right if you say that I should not call it AllPlayers ( or AllPlayers2 🤣).  I have to change my schema and collection in my next userstory because I need to store a string with the url for cloudinary in my object too. I plan to name the new schema Player (like suggested from you) and have a new collection players with the new data. I will then just delete the AllPlayers2 collection. But for this userstory I would like to let the code like this.

Reply:
I didn´t truly understand how the code works, that is why I just changed a few things and observed what happens 🤷🏽. Now I understand better how it works